2024-02-03 00:00:49,317 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-02-03 00:00:49,317 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Configure stats pid to 79316
2024-02-03 00:00:49,317 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Loading settings from /Users/vlados/.config/wandb/settings
2024-02-03 00:00:49,317 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Loading settings from /Users/vlados/Desktop/Projects/ai/projects/pet_projects/imsh/wandb/settings
2024-02-03 00:00:49,317 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-03 00:00:49,317 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-03 00:00:49,317 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-02-03 00:00:49,317 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-02-03 00:00:49,317 INFO    MainThread:79316 [wandb_init.py:_log_setup():526] Logging user logs to /Users/vlados/Desktop/Projects/ai/projects/pet_projects/imsh/wandb/run-20240203_000049-qe9z5deo/logs/debug.log
2024-02-03 00:00:49,317 INFO    MainThread:79316 [wandb_init.py:_log_setup():527] Logging internal logs to /Users/vlados/Desktop/Projects/ai/projects/pet_projects/imsh/wandb/run-20240203_000049-qe9z5deo/logs/debug-internal.log
2024-02-03 00:00:49,317 INFO    MainThread:79316 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x29ce2c370>
2024-02-03 00:00:49,317 INFO    MainThread:79316 [wandb_init.py:init():566] calling init triggers
2024-02-03 00:00:49,317 INFO    MainThread:79316 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'model': {'n_width': 20, 'n_coupling': 32, 'input_width': 1}, 'train': {'lr': 0.001, 'batch_size': 1000, 'epochs': 10, 'device': device(type='cpu')}}
2024-02-03 00:00:49,317 INFO    MainThread:79316 [wandb_init.py:init():616] starting backend
2024-02-03 00:00:49,317 INFO    MainThread:79316 [wandb_init.py:init():620] setting up manager
2024-02-03 00:00:49,318 INFO    MainThread:79316 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-02-03 00:00:49,318 INFO    MainThread:79316 [wandb_init.py:init():628] backend started and connected
2024-02-03 00:00:49,320 INFO    MainThread:79316 [wandb_run.py:_label_probe_notebook():1294] probe notebook
2024-02-03 00:00:49,320 INFO    MainThread:79316 [wandb_run.py:_label_probe_notebook():1304] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-02-03 00:00:49,320 INFO    MainThread:79316 [wandb_init.py:init():720] updated telemetry
2024-02-03 00:00:49,320 INFO    MainThread:79316 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-03 00:01:02,978 INFO    MainThread:79316 [wandb_run.py:_on_init():2254] communicating current version
2024-02-03 00:01:07,990 INFO    MainThread:79316 [wandb_run.py:_on_init():2263] got version response 
2024-02-03 00:01:07,990 INFO    MainThread:79316 [wandb_init.py:init():804] starting run threads in backend
2024-02-03 00:01:08,047 INFO    MainThread:79316 [wandb_run.py:_console_start():2233] atexit reg
2024-02-03 00:01:08,047 INFO    MainThread:79316 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-02-03 00:01:08,047 INFO    MainThread:79316 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-02-03 00:01:08,047 INFO    MainThread:79316 [wandb_run.py:_redirect():2178] Redirects installed.
2024-02-03 00:01:08,048 INFO    MainThread:79316 [wandb_init.py:init():847] run started, returning control to user process
2024-02-03 00:01:08,099 INFO    MainThread:79316 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-03 00:01:08,099 INFO    MainThread:79316 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-03 00:01:09,803 INFO    MainThread:79316 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-03 00:01:09,812 INFO    MainThread:79316 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-03 00:01:09,813 INFO    MainThread:79316 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-03 00:01:11,111 INFO    MainThread:79316 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-03 00:01:11,229 INFO    MainThread:79316 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-03 00:01:11,229 INFO    MainThread:79316 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-03 00:01:12,750 INFO    MainThread:79316 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-03 00:01:12,753 INFO    MainThread:79316 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-03 00:01:12,753 INFO    MainThread:79316 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-03 00:01:13,533 INFO    MainThread:79316 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-03 00:01:13,536 INFO    MainThread:79316 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-03 00:01:13,536 INFO    MainThread:79316 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-03 00:01:13,734 INFO    MainThread:79316 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-03 00:01:13,763 INFO    MainThread:79316 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-03 00:01:13,763 INFO    MainThread:79316 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-03 00:01:14,306 INFO    MainThread:79316 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-03 00:01:14,327 INFO    MainThread:79316 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-03 00:01:14,327 INFO    MainThread:79316 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-03 00:01:15,122 INFO    MainThread:79316 [wandb_init.py:_resume_backend():442] resuming backend
2024-02-03 00:01:15,131 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-02-03 00:01:15,131 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Configure stats pid to 79316
2024-02-03 00:01:15,131 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Loading settings from /Users/vlados/.config/wandb/settings
2024-02-03 00:01:15,131 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Loading settings from /Users/vlados/Desktop/Projects/ai/projects/pet_projects/imsh/wandb/settings
2024-02-03 00:01:15,131 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-03 00:01:15,131 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-03 00:01:15,131 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-02-03 00:01:15,131 INFO    MainThread:79316 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-02-03 00:01:15,132 INFO    MainThread:79316 [wandb_init.py:_log_setup():526] Logging user logs to /Users/vlados/Desktop/Projects/ai/projects/pet_projects/imsh/wandb/run-20240203_000115-948lwzn4/logs/debug.log
2024-02-03 00:01:15,132 INFO    MainThread:79316 [wandb_init.py:_log_setup():527] Logging internal logs to /Users/vlados/Desktop/Projects/ai/projects/pet_projects/imsh/wandb/run-20240203_000115-948lwzn4/logs/debug-internal.log
2024-02-03 00:01:15,132 INFO    MainThread:79316 [wandb_init.py:init():566] calling init triggers
2024-02-03 00:01:15,132 INFO    MainThread:79316 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'model': {'n_width': 20, 'n_coupling': 32, 'input_width': 1}, 'train': {'lr': 0.001, 'batch_size': 1000, 'epochs': 10, 'device': device(type='cpu')}}
2024-02-03 00:01:15,132 INFO    MainThread:79316 [wandb_init.py:init():591] re-initializing run, found existing run on stack: qe9z5deo
2024-02-03 00:01:15,133 INFO    MainThread:79316 [wandb_run.py:_finish():1962] finishing run ai-gigachads/imsh/qe9z5deo
2024-02-03 00:01:15,133 ERROR   MainThread:79316 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-02-03 00:01:15,134 INFO    MainThread:79316 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-03 00:01:15,134 INFO    MainThread:79316 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-02-03 00:01:15,134 INFO    MainThread:79316 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-02-03 00:01:15,134 INFO    MainThread:79316 [wandb_run.py:_restore():2185] restore
2024-02-03 00:01:15,134 INFO    MainThread:79316 [wandb_run.py:_restore():2191] restore done
2024-02-03 00:01:26,680 INFO    MainThread:79316 [wandb_run.py:_footer_history_summary_info():3857] rendering history
2024-02-03 00:01:26,681 INFO    MainThread:79316 [wandb_run.py:_footer_history_summary_info():3889] rendering summary
2024-02-03 00:01:26,688 INFO    MainThread:79316 [wandb_run.py:_footer_sync_info():3816] logging synced files
